packages/backend/src/misc/is-instance-muted.ts

```diff
@@ -0,0 +1,7 @@
+export function isInstanceMuted(note: any, muted_instances: Set<string>): boolean {
```

Comment:
```suggestion
import { Packed } from "./schema";

export function isInstanceMuted(note: Packed<'Note'>, muted_instances: Set<string>): boolean {
```

https://github.com/misskey-dev/misskey/pull/7712/files/9eed7816169f46babb34c69145a8c61e9b371e7f#r702389375  
Add type definition